### DISCUSSION TITLE
Regarding the experiment protocol

### DISCUSSION THREAD

**Discussion by manjunathnilugal** [pybamm-team/PyBaMM#2023] [Q&A] [answered]
Dear pybamm team, 

I am trying to simulate the experiment beginning with charging protocol. 
Is it possible to simulate the experiment **initially with charging** then discharging ?
If my understanding is wrong, kindly let me know or is there any method to start the simulation with charging.

```
import pybamm
model = pybamm.lithium_ion.DFN(options = {'particle':'Fickian diffusion'})
param = pybamm.ParameterValues("Chen2020")
experiment = pybamm.Experiment([("Charge at 1C until 4V")])
safe_solver = pybamm.CasadiSolver(mode = "safe", rtol=1e-8, atol=1e-8, dt_max=1)
sim = pybamm.Simulation(model, parameter_values = param, solver = safe_solver, experiment = experiment)
solution = sim.solve(calc_esoh=False)
sim.plot()
```
regards,
Manjunath

**Comment by valentinsulzer** [comment, marked as answer]
Yes, you can do this, but since the default parameters are for a fully charged battery you will also need to adjust the parameters (initial concentrations). The easiest way to do this is to pass `initial_soc=0` to `sim.solve()`.
